integration-cli/docker_api_logs_test.go

```diff
@@ -85,3 +86,94 @@ func (s *DockerSuite) TestLogsAPIContainerNotFound(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusNotFound)
 }
+
+func extractBody(c *check.C, url string) []string {
+	res, body, err := request.Get(url)
+	defer body.Close()
+	c.Assert(err, checker.IsNil)
+	c.Assert(res.StatusCode, checker.Equals, http.StatusOK)
+
+	reader := bufio.NewReader(body)
+	bytes, err := ioutil.ReadAll(reader)
+	c.Assert(err, checker.IsNil)
+
+	var out []string
+	for _, line := range strings.Split(string(bytes), "\n") {
+		parts := strings.Split(line, "$")
+		if len(parts) > 1 {
+			out = append(out, parts[1])
+		}
+	}
+	return out
+}
+
+func (s *DockerSuite) TestLogsAPIUntilFutureFollow(c *check.C) {
+	// TODO Windows TP5 - Figure out why this test is so flakey. Disabled for now.
+	// Also disabled in https://github.com/moby/moby/blob/eeaa6c96d83575da765dfc626c2c73d8a29dda32/integration-cli/docker_cli_logs_test.go#L182-L183
+	testRequires(c, DaemonIsLinux)
+
+	until := time.Now().Unix() + 5
```

Comment:
This needs to get the time one the daemon, not the local time.
Quite likely the issue with TP5.